egi/api_resources/gamelisting/checkin.py

```diff
@@ -103,8 +103,6 @@ def post(self):
         if not args.total_account_count \
            and args.total_player_count:
             args.total_account_count = args.total_player_count
-        # now that we're done with the old args, remove them
-        del args.total_player_count, args.connect_player_count
 
```

Comment:
Since it doesn't cause an error to not delete these keys, but does cause an error to delete them if they're not there...we won't delete them. One could also check to see if they exist and delete them if so...

Reply:
Except I'm wrong, and it does cause errors. Please hold.

Reply:
Take your time and let me know when you're ready for another round of review. I've got to step away from my compy at home that has this all set up, but can take another look tonight or this weekend.

Reply:
I will let you know - the Namespace is being very odd. It reports True to `hasattr(args, 'connected_player_count')` for instance, but `delattr(args, 'connected_player_count')` fails with an exception `Namespace has no attribute 'connected_player_count'`

Reply:
how about just `del args.connected_player_count`?

Reply:
It throws an exception if we don't provide connected_player_count. O.o

Reply:
I've "solved" this by setting the two optional args to None if they aren't. `if not None` could probably just be `if connected_player_count`, of course.